package.json

```diff
@@ -104,10 +104,10 @@
     "auto-require-webpack-plugin": "1.0.1",
     "chalk": "2.4.1",
     "clean-webpack-plugin": "0.1.17",
-    "copy-webpack-plugin": "4.5.2",
     "cli-columns": "3.1.2",
     "copy-webpack-plugin": "4.5.2",
     "css-loader": "0.28.7",
+    "eventsource-polyfill": "^0.9.6",
```

Comment:
Any reason not to pin this dependency as we've done with the others?

Reply:
Ah thanks - just npm installed didn't I!